### PR TITLE
Handle transactions with zero eth transfer

### PR DESCRIPTION
**Pull request by nebolax**
Currently waiting for CI to check which tests need an update

### CODE REVIEW

File: rotkehlchen/tests/unit/decoders/test_main.py [hunk 1]
```diff
@@ -57,3 +69,39 @@ def test_decoders_initialization(evm_transaction_decoder):
...
+    https://etherscan.io/tx/0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f
+    """
+    tx_hash = deserialize_evm_tx_hash('0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f')  # noqa: E501
+    # We don't need any events here, we just check that no errors occure during decoding
+    events = get_decoded_events_of_transaction(
+        ethereum_manager=ethereum_manager,
+        database=database,
+        msg_aggregator=function_scope_messages_aggregator,
+        tx_hash=tx_hash,
+    )
```
Comment (yabirgb) [inline rotkehlchen/tests/unit/decoders/test_main.py:90]: Please don't query the transaction. Instead add the info manuall to the database and decode it like we do in tests for other decoders. This add an extra call to etherscan that can fail, is slow and would contribute to rate limits

File: rotkehlchen/chain/ethereum/decoding/decoder.py [hunk 2]
```diff
@@ -431,10 +431,11 @@ def _maybe_decode_simple_transactions(
             ))
             return events
 
-        if amount == ZERO:
-            return events
-
         preposition = 'to' if verb == 'Send' else 'from'
+        if amount == ZERO:
+            notes = 'There was no ETH transfer in this transaction'
```
Comment (yabirgb) [inline rotkehlchen/chain/ethereum/decoding/decoder.py:436]: This message seems confusing at least to me.

```
The amount of ETH transfered in this transaction was 0.
```

From the curren note I understand that all transactions should have an eth transfer and this one doesn't have it

File: rotkehlchen/tests/unit/decoders/test_main.py [hunk 3]
```diff
@@ -57,3 +69,39 @@ def test_decoders_initialization(evm_transaction_decoder):
...
+
+@pytest.mark.parametrize('ethereum_accounts', [['0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045']])  # noqa: E501
+def test_no_logs_and_zero_eth(
+        database,
+        ethereum_manager,
+        function_scope_messages_aggregator,
+):
+    """Data taken from
+    https://etherscan.io/tx/0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f
+    """
```
Comment (yabirgb) [inline rotkehlchen/tests/unit/decoders/test_main.py:82]: I belive this is the style @LefterisJP is following in the comments

```suggestion
    """
    Data taken from
    https://etherscan.io/tx/0x9a95424c48d36bb2f60fb7684a1068c08ec643c64144e7cdfbe5fb3fc820aa7f
    """
```

File: rotkehlchen/chain/ethereum/decoding/decoder.py (hunk 2 above)
Comment (nebolax) [reply rotkehlchen/chain/ethereum/decoding/decoder.py:436]: Yeah, makes sense. Do you have any ideas for this note?

File: rotkehlchen/tests/unit/decoders/test_main.py (hunk 1 above)
Comment (nebolax) [reply rotkehlchen/tests/unit/decoders/test_main.py:90]: This is an okayish approach and used in some decoders tests (Lefteris also mentioned at some point that speed of writing decoders tests justifies longer tests running). But no problem, I can change it

File: rotkehlchen/tests/unit/decoders/test_main.py (hunk 3 above)
Comment (nebolax) [reply rotkehlchen/tests/unit/decoders/test_main.py:82]: Yeah, alright. I just copy-pasted in from another decoder test

File: rotkehlchen/tests/unit/decoders/test_main.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/tests/unit/decoders/test_main.py:90]: I don't believe is okey. You can try it and see how the time increases considerably making remote calls to etherscan vs hard coding them. Also in the places where someone used them we have
image

File: rotkehlchen/chain/ethereum/decoding/decoder.py (hunk 2 above)
Comment (yabirgb) [reply rotkehlchen/chain/ethereum/decoding/decoder.py:436]: The one I wrote is the one that I believe is the clearest. Maybe

```
This transaction included a transfer of ETH with value 0
```

Comment (nebolax) [reply rotkehlchen/chain/ethereum/decoding/decoder.py:436]: Ah, sorry, thought you just pasted my version in the comment. But hm... there is no eth transfer in such transactions, so sounds more confusing to me than the original version

Comment (yabirgb) [reply rotkehlchen/chain/ethereum/decoding/decoder.py:436]: So maybe "This transactions didn't contain any trackable event"

Comment (nebolax) [reply rotkehlchen/chain/ethereum/decoding/decoder.py:436]: I guess would be a great option if it was the only event in transaction. But the problem is that for consistency, info about amount of eth "transfered" will be present in all transactions

Comment (LefterisJP) [reply rotkehlchen/chain/ethereum/decoding/decoder.py:436]: A 0 value transfer is fine. Transferred 0 ETH to XXX.

**Review by LefterisJP — APPROVED** [review]
Some minor comments that can be addressed in another PR. I think both tests are due to timeouts, so will merge.

**Pull request merged**